index.html

```diff
@@ -142,17 +142,20 @@ <h2>Skills</h2>
       <div class="modal" id="modal-id"></div>
     </div>
   </main>
-    <footer id="contact-me">
+    <footer id="contact-me-footer">
       <div class="footer-wrapper">
         <div class="chat-text">
           <h2>
           I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
           </h2>
         </div>
-        <form action="https://formspree.io/f/xoqrojkw" method="POST">
+        <form id="contact-me" action="https://formspree.io/f/xoqrojkw" method="POST">
           <div class="input-field">
-            <input type="text" name="name" maxlength="30" class="text-input" placeholder="Full name" required>
-            <input type="email" name="Email" class="text-input" placeholder="Email address" required>
+            <input type="text" name="name" id="user-name" maxlength="30" class="text-input" placeholder="Full name" required>
+            <input type="email" id="user-email" name="Email" class="text-input" placeholder="Email address" required>
+            <div class="error-message">
```

Comment:
- [ ] Great job adding your error message for "Emails written in Capital Letter", however pls kindly add your error message close to your submit button and not above the textarea like you have done.

Reply:
Thanks for the feedback I have edited my error message.